dev-scripts/deploy_custom_rh-che.sh

```diff
@@ -197,7 +197,7 @@ curl -L0fs $DEPLOY_POSTGRES_CONFIG_URL -o che-init-image-stream.yaml 2>1 > /dev/
 # GET POSTGRES CONFIGS
 mkdir postgres 2>1 > /dev/null
 cd postgres || exit 1
-export IMAGE_POSTGRES="centos/postgresql-96-centos7"
+export IMAGE_POSTGRES="eclipse/che-postgres"
```

Comment:
@ScrewTSW I had to use this image to make it run correctly with the last upstream master.

If you fix this in another PR, I'll change it here and will rebase my PR on your.

Reply:
I don't think it's necessary to open a new PR just for this change. If the image works better, then let's use that one :)

Reply:
Radim suggested and I think he's right, the issues is with "You do not have access to this namespace" which is probably incorrectly handled by the CLI `origin-clients` package.

Basically, you cannot have namespaces with the same name, and it failed for you, because I already had the namespace created in my account, tied to my user.

Is there a way that origin would allow to create multiple namespaces with the same name?